src/community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/request/TemplatePathVisitor.java

```diff
@@ -29,15 +29,15 @@
  * This visitor search for a Filter in {@link TemplateBuilder} tree using the path provided as a
  * guideline.
  */
-public class JSONPathVisitor extends DuplicatingFilterVisitor {
+public class TemplatePathVisitor extends DuplicatingFilterVisitor {
```

Comment:
Looking into the class contents, there are still mentions of JSON though, and at least one bit that grabs the JsonNode from a static builder (is that working with an XML template?).

Reply:
fixed